build/tools/Automation/DotNet/DotNet.cs

```diff
@@ -127,6 +131,27 @@ public override async Task<List<PlatformConstant>> GetPlatformConstantsAsync(str
             return platformConstants;
         }
 
+        public override async Task CacheSdkVersionsAsync(string platform)
+        {
+            string url = $"https://oryxsdks.blob.core.windows.net/{platform}" +
+            "?restype=container&comp=list&include=metadata";
```

Comment:
Decided to check prod SDK account instead of dev. This will help us re-generate a new fresh PR with updated main changes. If integrating new SDKs ever takes longer than usuals without having to delete new SDKs from dev storage account.

Reply:
I moved the SDK CDN base URL into the `constants.yaml`. I kept prod since there is value in being explicit.